clang/test/ParserOpenACC/parse-constructs.c

```diff
@@ -94,6 +94,37 @@ void func() {
 #pragma acc kernels loop
   for(;;){}
 
+  int i = 0, j = 0, k = 0;
+  // expected-error@+2{{missing OpenACC 'atomic-clause'; expected 'read', 'write', 'update', or 'capture'}}
+  // expected-warning@+1{{OpenACC directives not yet implemented, pragma ignored}}
+#pragma acc atomic
```

Comment:
It is allowed by the standard, means same as atomic update

Reply:
Ah!  I'd not caught that on my read through the standard.  Thanks!